### DISCUSSION TITLE
Hosting on fly.io - How does it scale?

### DISCUSSION THREAD

**Discussion by Stefan2016** [zauberzeug/nicegui#1790] [Q&A]
### Question

Dear all,

I was reading the wiki regarding fly.io hosting. Thanks for that.

How does that work when the app gets many concurrent accesses, does it scale automatically to multiple machines using multiple docker containers, load balancers and similar?

I am asking as I especially looking for such an option and I am not quite sure if I get the fly.io offer right.

Link to the wiki: https://github.com/zauberzeug/nicegui/wiki/fly.io-Deployment

**Comment by rodja** [comment]
Have you read https://fly.io/docs/apps/scale-count/ which describes horizontal scaling (i.e. creating multiple machines)? In combination with https://fly.io/docs/apps/autostart-stop/ the load balancer will automatically start and stop machines according to demand.
